easybuild/easyconfigs/p/PyImageJ/PyImageJ-1.3.1-foss-2021a.eb

```diff
@@ -0,0 +1,60 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonBundle'
+
+name = 'PyImageJ'
+version = '1.3.1'
+
+homepage = "https://pyimagej.readthedocs.io/"
+description = """PyImageJ provides a set of wrapper functions for integration between ImageJ2 and Python. It also
+ supports the original ImageJ API and data structures. A major advantage of this approach is the ability to combine
+ ImageJ and ImageJ2 with other tools available from the Python software ecosystem, including NumPy, SciPy,
+ scikit-image, CellProfiler, OpenCV, ITK and many more."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('Java', '11', '', SYSTEM),
+    ('Maven', '3.6.3', '', SYSTEM),
+    ('matplotlib', '3.4.2'),
+    ('Pillow', '8.2.0'),
+    ('scikit-image', '0.18.3'),
+    ('SciPy-bundle', '2021.05'),
+    ('xarray', '0.19.0'),
```

Comment:
@branfosj Never used this package, but does it make sense to add a dependency on `ImageJ`?

Reply:
A dependency on `ImageJ` would only get used if:
1. each user specifies to use that version of `ImageJ` every time they initialise
1. we made an edit to the code edits to default to that install

I am not sure the first option is sensible and that the second one is worthwhile.

Reply:
I agree, a dependency on `ImageJ` will be too much of a constrain and it deviates significantly from the default workflow with `PyImageJ`. 

@verdurin what do you think?

Reply:
Sounds fine to me.